package.json

```diff
@@ -119,6 +119,7 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.901.5",
     "@angular-devkit/build-ng-packagr": "~0.901.5",
+    "@angular/bazel": "^9.1.5",
```

Comment:
That's the one that is being deprecated, right?
https://github.com/angular/angular/pull/37190

Why do we need it? 🙂

Reply:
It's needed in the current toolchain we use for the `ng_module`, and `ng_package` rules. 
https://github.com/ngrx/platform/blob/master/tools/defaults.bzl#L5